shared/js/textChatStore.js

```diff
@@ -148,13 +155,24 @@ loop.store.TextChatStore = (function() {
     },
 
     /**
-     * Handles sending of a chat message.
+     * Handles sending of a chat message.  Adds a name (XXX yuck, this
+     * shouldn't be so side-effecty)
```

Comment:
I had to undo the displayName check, as it turns out that the tile code calls this, and it doesn't have any displayName.